plugins/Widgetize/vue/src/ExportWidget/ExportWidget.vue

```diff
@@ -0,0 +1,169 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <div class="widgetize">
+    <div v-content-intro>
+      <h2>
+        <EnrichedHeadline>{{ title }}</EnrichedHeadline>
+      </h2>
+      <p>With Matomo, you can <a
```

Comment:
Would you mind moving all the text to translation files? Otherwise guess we can also create a small issue to do that another time.

Reply:
That's enough text for me to consider the change out of scope.

Reply:
Ok. Will create a follow up issue, once this is merged.